bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -449,14 +467,20 @@ async def _nominate_context_callback(self, interaction: discord.Interaction, mes
                 ":white_check_mark: Existing nomination updated",
                 ephemeral=True
             )
+
+            await self.maybe_relay_update(
+                message.author.id,
+                f":new: <@{interaction.user.id}> has attached {message.jump_url} to their nomination"
```

Comment:
Maybe use `.mention` for consistency

Reply:
Good call, fixed in https://github.com/python-discord/bot/compare/7df00e07417b06f2f5646698fc5049894a03ebcf..e710d9f2741dfe5c3b9b34cd1226a0cec9088820